providers/base/bin/virtualization.py

```diff
@@ -734,19 +734,20 @@ def __init__(self, template=None, image=None):
         self.default_remote = "ubuntu:"
         self.os_version = get_release_to_test()
 
-    def run_command(self, cmd):
+    def run_command(self, cmd, error_msgs=True):
```

Comment:
I would change this parameter to `log_stderr` because it avoids the `logger.error` call but also the `logger.debug` further below